mix.exs

```diff
@@ -36,8 +36,9 @@ defmodule ExMachina.Mixfile do
     [
       {:ex_doc, "~> 0.14", only: :dev},
       {:earmark, ">= 0.0.0", only: :dev},
-      {:ecto, "~> 2.1", optional: true},
-      {:postgrex, ">= 0.0.0", only: [:test]},
+      {:ecto_sql, "~> 3.0.0-pre", optional: true},
```

Comment:
`ecto ~> 2.1 or ~3.0` would work too

Reply:
agreed, that would definitely be preferred and it matches phoenix_ecto: https://github.com/phoenixframework/phoenix_ecto/blob/v3.5.0/mix.exs#L40